docs/upgrades/unreleased.md

```diff
@@ -1,179 +1,207 @@
+
+  ```
+      new GraphQLDictionaryService({
+        siteName,
+        clientFactory,
+        .....
+        useSiteQuery: true,
+      })
+
+  ```
+
+- We have introduced a new configuration option, `pagesEditMode`, in the `\src\pages\api\editing\config.ts` file to support the new editing metadata architecture for Pages (XMCloud). This option allows you to specify the editing mode used by Pages. It is set to `metadata` by default. However, if you are not ready to use a new integration and continue using the existing architecture, you can explicitly set the `pagesEditMode` to `chromes`.
+
+  ```ts
+      import { EditMode } from '@sitecore-jss/sitecore-jss-nextjs';
+
+      const handler = new EditingConfigMiddleware({
+      ...
+      pagesEditMode: EditMode.Chromes,
+      }).getHandler();
+  ```
+
+- Introduce a new _lib/graphql-editing-service.ts_ file to initialize a _graphQLEditingService_ to support a new Editing Metadata Mode. Can be done by adding this file from the latest version introduced in _nextjs-xmcloud_ base template.
+
+- Update _lib/page-props-factory/plugins/preview-mode_ plugin to support a new Editing Metadata Mode. Can be done by replacing this file with the latest version introduced in _nextjs-xmcloud_ base template.
+
+- To support editing for fields in Pages, the new editing metadata architecture relies on the new metadata property 'field.metadata' (instead of on 'field.editable', which won't be used in this scenario). If you are using the new editing arhitecture in Pages (EditMode.Metadata) and have custom field component that manipulates or relies on 'field.editable' in some way, it may need to be reworked. Experience Editor still relies on 'field.editable', so it needs to be supported. See example below from SXA's Banner component:
+
+  ```ts
+      import { useSitecoreContext, EditMode } from '@sitecore-jss/sitecore-jss-nextjs';
+      ...
+      export const Banner = (props: ImageProps): JSX.Element => {
+          const { sitecoreContext } = useSitecoreContext();
+          const isMetadataMode = sitecoreContext?.editMode === EditMode.Metadata;
+          ...
+          const modifyImageProps = !isMetadataMode
+          ? {
+              ...props.fields.Image,
+              editable: props?.fields?.Image?.editable
+              ?.replace(`width="${props?.fields?.Image?.value?.width}"`, 'width="100%"')
+              .replace(`height="${props?.fields?.Image?.value?.height}"`, 'height="100%"'),
+          }
+          : { ...props.fields.Image };
+          ...
+      }
+      ...
+  ```
+
+- Jss now supports component level personalization for XM Cloud. See below code from personalize.ts. _personalizeData_ contains _componentVariantIds_ property which has the personalized components. It can be passed to the _personalizeLayout_ function to apply the personalization to the _layoutData_:
```

Comment:
I think it's better to outline the goal more directly and in a less verbose way (i.e. "To enable AB testing support in JSS, ensure componentVariantIds are passed to personalizeLayout"). Official docs will deal with the rest.